### PR TITLE
Show a warning when unsupported browser is detected.

### PR DESCRIPTION
**Pull request by kenzieschmoll**
Happy to change wording. CC @sfshaza2 
Screen Shot 2019-03-21 at 1 06 56 PM
Screen Shot 2019-03-21 at 1 07 17 PM

### CODE REVIEW

**Comment by jacob314** [comment]
**Lets make this a RED background. Things will be broken.**

File: packages/devtools/web/main.dart [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 import 'package:devtools/src/framework/framework_core.dart';
 import 'package:devtools/src/main.dart';
+import 'package:platform_detect/platform_detect.dart';
```
Comment (jacob314) [inline packages/devtools/web/main.dart]: Good call finding a package that already does this!

Comment (jacob314) [reply packages/devtools/web/main.dart]: that package is maintained by a company that is a long time dart customer so is quite safe for us to use. if the package was maintained by someone random user we would need to be more careful.

File: packages/devtools/web/main.dart [hunk 2]
```diff
@@ -20,5 +21,10 @@ void main() {
     }
   });
 
+  if (!browser.isChrome) {
+    framework.showWarning('WARNING: Unsupported browser; DevTools is only '
```
Comment (jacob314) [inline packages/devtools/web/main.dart]: Users are unfortunately blind to warning messages.
Make this the only content that is displayed and require the user to click somewhere to continue to use the app in an unsupported browser or don't even give users that option as things are likely badly broken.
In the spirit of error message improvement, tweak the message to something like:
ERROR: You are running devtools with $browserName but DevTools only runs on Chrome.

Reopen this url in a Chrome browser to use devtools.

<link>Continue to use devtools in this unsupported browser</link>

What I like about this general form is the first line covers the problem and indicates the solution. I'm not great at writing error messages so feel free to weak or ignore my suggestion.

**Comment by kenzieschmoll** [comment]
Updated screenshots:
Screen Shot 2019-03-21 at 2 31 31 PM
Screen Shot 2019-03-21 at 2 31 52 PM

**Comment by jacob314** [comment]
lgtm

**Comment by kenzieschmoll** [comment]
Fixes https://github.com/flutter/devtools/issues/66

**Pull request merged**